docs/migration-guide.md

```diff
@@ -24,7 +24,8 @@ To upgrade your Storybook:
 <CodeSnippets
   paths={[
     'common/storybook-upgrade-prerelease.npm.js.mdx',
-    'common/storybook-upgrade-prerelease.pnpm.js.mdx'
+    'common/storybook-upgrade-prerelease.pnpm.js.mdx',
+    'common/storybook-upgrade-prerelease.yarn.js.mdx'
```

Comment:
@jackw can you update not only the snippet's naming per #21918 but also the command you've used as well so that we provide the correct information?

Reply:
Ah ok, I missed that. I'll rebase, fix conflicts and update the commands as requested. Thanks for the feedback @jonniebigodes ! 🙏

Reply:
Actually, it looks like this is targetting `next` but the PRs mentioned are targeting `main`. 🤔

Reply:
Indeed you're right; those were targeting the `main` branch. But you don't need to worry about it. Once this is merged, it will be patched up to the `main` branch.

Reply:
Ah ok cool. I'll sort it out shortly. Thanks for getting back to me so quickly.